docs_src/getting-started/native/Ch-BuildRunOnLinuxDistro.md

```diff
@@ -0,0 +1,376 @@
+
+Enter the `edgex-ui-go` folder and issue the `make build` command as shown below.
+
+![image](BuildGUI.png)
+
+## Run EdgeX
+
+Provided everything built correctly and without issue, you can now start your EdgeX services one at a time.  First make sure Redis Server is running.  If Redis is not running, start it before the other services.  If it is running, you can start each of the EdgeX services **in order** as listed below.
+
+### Start Consul
+
+Start Consul Agent with the following command.
+
+``` Shell
+nohup consul agent -ui -bootstrap -server -client 0.0.0.0 -data-dir=tmp/consul &
+```
+
+The `nohup` is used to execute the command and ignore all SIGHUP (hangup) signals.  The `&` says to execute the process in the background.  Both `nohup` and `&` will be used to run each of the services so that the same terminal can be used and the output will be directed to local nohup.out log files.
+
+If Consul is running correctly, you should be able to reach the Consul UI through a browser at http://(host address):8500
+
+![image](RunConsulUI.png)
+
+### Start Core Metadata
+
+Each of core and supporting EdgeX services are located in `edgex-go/cmd` under a subfolder by the service name.  In the first case, core-metadate is located in `edgex-go/cmd/core-metadata`.  Change directories to the core-metadata service subfolder and then run the executable found in the subfolder with `-cp` and `-registry` command line options as shown below.
+
+``` shell
+cd edgex-go/cmd/core-metadata/
+nohup ./core-metadata -cp=consul.http://localhost:8500 -registry &
+```
+
+The `-cp=consul.http://localhost:8500` command line parameter tells core-metadata to use Consul and where to find Consul running.  The `-registry` command line parameter tells core-metadata to use (and register with) the registry service.  Both of these command line parameters will be use when launching all EdgeX services.
+
+### Start the other Core and Supporting Services
+
+In a similar fashion, enter each of the other core and supporting service folders in `edgex-go/cmd` and launch the services.
+
+```Shell
+cd ../core-data
+nohup ./core-data -cp=consul.http://localhost:8500 -registry &
+cd ../core-command
+nohup ./core-command -cp=consul.http://localhost:8500 -registry &
+cd ../support-notifications/
+nohup ./support-notifications -cp=consul.http://localhost:8500 -registry &
+cd ../support-scheduler/
+nohup ./support-scheduler -cp=consul.http://localhost:8500 -registry &
+```
+
+!!! Tip
+    If you still have the Consul UI up, you should see each of the EdgeX core and supporting services listed in Consul's `Services` page with green check marks next to them suggesting they are running.
+
+    ![image](CoreSupportingServicesRunning.png)
```

Comment:
Verify that is is rendering. It doesn't in plain MD web viewer.

Reply:
It did for me Lenny.  It is an image inside the admonition so it requires to run with the Mkdocs server (not just MD web viewer).